bower.json

```diff
@@ -14,7 +14,7 @@
     "tests"
   ],
   "dependencies": {
-    "hello": "git@github.com:dan-silver/hello.js.git#master",
```

Comment:
Not sure if this is more a question for Kevin, but you're just changing an existing dependency, right? So I won't have to go through the longer process for when a new one is added?